apps/website/src/content/docs/datepicker.mdx

```diff
@@ -73,7 +75,7 @@ To improve user experience, date pickers support year skipping buttons. Clicking
   <AllExamples.DatePickerWithYearExample client:load />
 </LiveExample>
 
-## Usage
+## Standalone usage
 
 A date picker can be featured on your page in multiple ways. One can be added directly to your page so that it is visible as soon as the page loads. Additionally, a date picker can be added to a menu button on your page so that the user can toggle its visibility.
```

Comment:
This text is out-of-date with the example. It should be focused on explaining the standalone usage, and specifically mention the `applyBackground` prop as well.

```suggestion
The `DatePicker` does not always need to be used within its own popover. You can instead display it as part of a larger interface containing interconnected controls.

You can set `applyBackground={false}` to make the `DatePicker` blend into its surroundings. This is useful when the surrounding UI already brings its own background-color and border.
```

Reply:
Added these in the latest push.

Reply:
Missing second paragraph